README.md

```diff
@@ -398,6 +398,13 @@ You will need the following things properly installed on your computer.
 
 * `git clone https://github.com/ExpDev07/coronavirus-tracker-api.git`
 * `cd coronavirus-tracker-api`
+
+Now, make sure you're running python 3.8 and that your `pipenv` actually uses it! It may use packages from `$HOME/.local/lib/python3.7` even when started with `pipenv --python 3.8`. To combat this, (re)install it using:
+```
+/path/to/python3.8 -m pip install pipenv
```

Comment:
A better practice would be to use `pipx` or `brew` to install `pipenv`
https://pipxproject.github.io/pipx/

Reply:
But then you need to install `pipx` in the first place, which is yet another thing which can go wrong. So when having the choice - to get X done:
1. you need to have Y.
2. just use tools you already have.

I prefer the 2., unless given sufficiently strong arguments for 1.

Anyway I'm not much of a python developer, and I don't really want to become one, so it's up to you - change it the way you like. That applies to both or your remarks.

Reply:
Arguments about the best way to install `pipenv` aside...
What you've written here is factually incorrect.
You don't need to re-install `pipenv` in order to have it create a `3.8` virtual environment.
`pipenv` installed via `3.6` can still create and use a `3.8` environment as long as `3.8` is installed and on the `PATH`
https://pipenv.readthedocs.io/en/latest/basics/#specifying-versions-of-python

You don't need to include detailed instructions about `brew` or `pipx` installations.
Something simple with links to more detailed instructions should be sufficient.

1. Make sure you have [`python3.8` installed and on your `PATH`](https://docs.python-guide.org/starting/installation/).
2. [Install the `pipenv` dependency manager](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv)
   * with pip/pip3 directly `$ pip install --user pipenv`
   * with Homebrew/Linuxbrew `$ brew install pipenv`
   *  with [pipx](https://pipxproject.github.io/pipx/) `$ pipx install pipenv`